IndexHelper.js

```diff
@@ -43,7 +47,16 @@ export class IndexHelper {
       throw new TypeError(
         '"attribute" must be a string or an array of strings.');
     }
-    attribute.forEach(x => this.indexes.set(x, unique));
+    attribute.forEach(x => {
+      const {scope} = this._formatAttribute(x);
+      if(!(scope &&
+        typeof scope === 'string' &&
+        VALID_SCOPES.includes(scope))) {
+        throw new TypeError(
+          '"scope" must be a string with value "content" or "meta"');
```

Comment:
No one will know what `scope` is since it isn't passed in ... we could say that the "attribute prefix" must be `content` or `meta` and hopefully that provides better semantics to the external user.